iam-login-service/src/main/java/it/infn/mw/iam/config/oidc/OidcConfiguration.java

```diff
@@ -232,4 +251,9 @@ UserInfoFetcher userInfoFetcher() {
   OidcTokenRequestor tokenRequestor(RestTemplateFactory restTemplateFactory, ObjectMapper mapper) {
     return new DefaultOidcTokenRequestor(restTemplateFactory, mapper);
   }
+
+  @Override
+  public void configureTasks(ScheduledTaskRegistrar taskRegistrar) {
+    cleanupScheduler.scheduleCleanupTask(taskRegistrar, jitProperties, accountService, "OIDC");
```

Comment:
Both this Configuration class and the SAML one schedule in practice the same cleanup task which deletes users independently from the fact they have been created through SAML or OIDC. Probably we should move a unique task creation/configuration inside TaskConfig class

Reply:
We decided to delete the cleanup task for now, as it is unused.